staging/src/k8s.io/dynamic-resource-allocation/resourceslice/resourceslicecontroller.go

```diff
@@ -134,15 +168,51 @@ func StartController(ctx context.Context, kubeClient kubernetes.Interface, drive
 		defer logger.V(3).Info("Stopping")
 		c.run(ctx)
 	}()
-
-	// Sync each pool once.
-	for poolName := range resources.Pools {
-		c.queue.Add(poolName)
-	}
-
```

Comment:
Why is this no longer needed?

Reply:
Edit: looks like this was just moved here:
https://github.com/kubernetes/kubernetes/pull/128275/files#diff-af98706b50db43cf2d25a2e2f986e2a8dc7e3d624d16e7197afe67e9e20bb8a7R299-R304